docs/guides/build-it-yourself/achieve-compliance/deployment-walkthrough/manual_steps.md

```diff
@@ -0,0 +1,129 @@
+---
+pagination_label: Deployment Walkthrough
+---
+
+# Manual steps
+
+Now that your infrastructure is up and running, there are some manual steps necessary before the resources meet the CIS AWS Benchmark recommendations.
+
+## Revoke Gruntwork's access
+
+Now that your infrastructure is deployed, Gruntwork doesn't need access to it anymore. Use the `gruntwork` CLI to revoke the access, [following these steps](https://github.com/gruntwork-io/gruntwork#revoking-access-to-aws):
+
+```bash
+gruntwork aws revoke \
+            --account "dev" \
+            --account "stage" \
+            --account "prod" \
+            --account "security" \
+            --account "shared" \
+            --account "logs"
+```
+
+To revoke Gruntwork's access from the "current" account—the one you are authenticated to in the terminal—use the name
+`__current__` (i.e., `--account "__current__"`).
+
+
+**Important**: Never use the `AdministratorAccess` AWS managed policy with any users, groups, or roles. It gives full access to all resources. Instead, use a policy that allows full IAM privileges (e.g. `iam:*`) on all resources. Our Landing Zone solution created a `iam-admin` group in the Security account.
```

Comment:
Does this section need its own header? Is this a CIS-Benchmark requirement or a Gruntwork best-practice recommendation? 

If we're recommending something to do with `IAM` permissions, we should probably give them some working IAM policy examples to illustrate - e.g. show the full tested JSON that they can use.

Reply:
Also, this sentence feels like it's starting an incomplete thought: 
> Our Landing Zone solution created a `iam-admin` group in the Security account.

Are we suggesting that customers use this IAM group for all their users going forward? or only for certain users that need administrator privileges?

Reply:
> Does this section need its own header? Is this a CIS-Benchmark requirement or a Gruntwork best-practice recommendation?

This is required for CIS, because without removing GW's access, the check for "no has uses `*:*` access" fails.

But customers should revoke access from GW after deploying their infra, so this duplicated in both guides.

 > Are we suggesting that customers use this IAM group for all their users going forward? or only for certain users that need administrator privileges?
 
The suggestion was for the customer to use this instead of the AWS-managed policy. I will improve the wording.